src/main/java/com/godaddy/commerce/migrator/handlers/helpers/BatcherHelper.java

```diff
@@ -1,81 +1,81 @@
 package com.godaddy.commerce.migrator.handlers.helpers;
 
+import static com.godaddy.commerce.migrator.models.Constants.ENTITLEMENTS;
+import static com.godaddy.commerce.migrator.models.Constants.LEGACY_FULFILLMENT;
+import static com.godaddy.commerce.migrator.models.Constants.SHOPPERS;
 import static java.lang.String.format;
 
 import com.godaddy.commerce.core.errors.ServiceException;
 import com.godaddy.commerce.core.http.HttpVerbResult;
```

Comment:
> [<img alt="mtbennett" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.secureserver.net/mtbennett) **Authored by [mtbennett](https://github.secureserver.net/mtbennett)**
_<time datetime="2021-02-03T17:54:30Z" title="Wednesday, February 3rd 2021, 10:54:30 am -07:00">Feb 3, 2021</time>_

---


Necessary for the `PREFER_LOCAL_DNS` suggestions below.

```suggestion
import com.godaddy.commerce.core.http.HttpVerbOption;
import com.godaddy.commerce.core.http.HttpVerbResult;
```